shinylive_lock.json

```diff
@@ -481,133 +455,24 @@
-    "url": "https://files.pythonhosted.org/packages/d1/1c/5aeb94aa980da111e4fd0c0fbe5ad95ed5bf9bd957f8e2a6178b85ff4da8/uc_micro_py-1.0.2-py3-none-any.whl",
+    "version": "1.0.3",
+    "filename": "uc_micro_py-1.0.3-py3-none-any.whl",
+    "sha256": "db1dffff340817673d7b466ec86114a9dc0e9d4d9b5ba229d9d60e5c12600cd5",
+    "url": "https://files.pythonhosted.org/packages/37/87/1f677586e8ac487e29672e4b17455758fce261de06a0d086167bb760361a/uc_micro_py-1.0.3-py3-none-any.whl",
     "depends": [],
     "imports": [
       "uc_micro"
     ]
   },
-  "tzdata": {
-    "name": "tzdata",
-    "version": "2023.3",
-    "filename": "tzdata-2023.3-py2.py3-none-any.whl",
-    "sha256": "7e65763eef3120314099b6939b5546db7adce1e7d6f2e179e3df563c70511eda",
-    "url": "https://files.pythonhosted.org/packages/d5/fb/a79efcab32b8a1f1ddca7f35109a50e4a80d42ac1c9187ab46522b2407d7/tzdata-2023.3-py2.py3-none-any.whl",
-    "depends": [],
-    "imports": [
-      "tzdata"
-    ]
-  },
-  "matplotlib-inline": {
-    "name": "matplotlib-inline",
-    "version": "0.1.6",
-    "filename": "matplotlib_inline-0.1.6-py3-none-any.whl",
-    "sha256": "f1f41aab5328aa5aaea9b16d083b128102f8712542f819fe7e6a420ff581b311",
-    "url": "https://files.pythonhosted.org/packages/f2/51/c34d7a1d528efaae3d8ddb18ef45a41f284eacf9e514523b191b7d0872cc/matplotlib_inline-0.1.6-py3-none-any.whl",
-    "depends": [
-      {"name": "traitlets", "specs": []}
-    ],
-    "imports": [
-      "matplotlib_inline"
-    ]
-  },
-  "prompt-toolkit": {
-    "name": "prompt-toolkit",
-    "version": "3.0.41",
-    "filename": "prompt_toolkit-3.0.41-py3-none-any.whl",
-    "sha256": "f36fe301fafb7470e86aaf90f036eef600a3210be4decf461a5b1ca8403d3cb2",
-    "url": "https://files.pythonhosted.org/packages/1f/9d/be9b01085bbd67a71c4b6aa02518fade8104e7a2224e5de5e947811d7176/prompt_toolkit-3.0.41-py3-none-any.whl",
-    "depends": [
-      {"name": "wcwidth", "specs": []}
-    ],
-    "imports": [
-      "prompt_toolkit"
-    ]
-  },
-  "stack-data": {
-    "name": "stack-data",
-    "version": "0.6.3",
-    "filename": "stack_data-0.6.3-py3-none-any.whl",
-    "sha256": "d5558e0c25a4cb0853cddad3d77da9891a08cb85dd9f9f91b9f8cd66e511e695",
-    "url": "https://files.pythonhosted.org/packages/f1/7b/ce1eafaf1a76852e2ec9b22edecf1daa58175c090266e9f6c64afcd81d91/stack_data-0.6.3-py3-none-any.whl",
-    "depends": [
-      {"name": "executing", "specs": [[">=", "1.2.0"]]},
-      {"name": "asttokens", "specs": [[">=", "2.1.0"]]},
-      {"name": "pure-eval", "specs": []}
-    ],
-    "imports": [
-      "stack_data"
-    ]
-  },
-  "idna": {
-    "name": "idna",
-    "version": "3.6",
-    "filename": "idna-3.6-py3-none-any.whl",
-    "sha256": "c05567e9c24a6b9faaa835c4821bad0590fbb9d5779e7caa6e1cc4978e7eb24f",
-    "url": "https://files.pythonhosted.org/packages/c2/e7/a82b05cf63a603df6e68d59ae6a68bf5064484a0718ea5033660af4b54a9/idna-3.6-py3-none-any.whl",
-    "depends": [],
-    "imports": [
-      "idna"
-    ]
-  },
```

Comment:
I'm not sure why, but when I run `make update_packages_lock` locally, it wants to add back in a lot of these dependencies. It also wants to move shiny from 1.0.0 to 0.10.2.9000 (even after I `cd packages/py-shiny && make install`). Maybe I'm missing something to run `make update_packages_lock` locally?

Reply:
First check if any other local package submodules are out of sync with the specified commit, with `git status`.

To update the version of shiny (and others packages) to the one pointed to by the submodule, run `make submodules`.

Then try running `make update_packages_lock`

Reply:
Yea, the submodules are up-to-date, and I still get the differences described above.